packages/plugins/users-permissions/admin/src/hooks/useFetchRole/index.js

```diff
@@ -1,13 +1,16 @@
-import { useCallback, useReducer, useEffect } from 'react';
+import { useCallback, useReducer, useEffect, useRef } from 'react';
 import { request, useNotification } from '@strapi/helper-plugin';
 import reducer, { initialState } from './reducer';
 import pluginId from '../../pluginId';
 
 const useFetchRole = id => {
   const [state, dispatch] = useReducer(reducer, initialState);
   const toggleNotification = useNotification();
+  const isMounted = useRef(null);
```

Comment:
This solution is taken from [a blog post I found](https://www.debuggr.io/react-update-unmounted-component/). It seemed to be necessary to make Jest happy. I wish I could have avoided it though